test/models/alias.ts

```diff
@@ -1,12 +1,12 @@
-import { getModelForClass, prop } from '../../src/typegoose';
+import { getModelForClass, prop } from '../../src/typegoose'
 
 export class Alias {
   @prop({ required: true })
-  public normalProp!: string;
+  public normalProp!: string
 
   @prop({ required: true, alias: 'aliasProp' })
-  public alias!: string;
-  public aliasProp!: string; // its just for type completion
+  public alias!: string
+  public aliasProp!: string // its just for type completion
 }
 
-export const AliasModel = getModelForClass(Alias);
+export const AliasModel = getModelForClass(Alias)
```

Comment:
why did this get changed?

Reply:
Made a save on the file to check something and my VSCode changed the styling. I forgot to correct it. 

Scott